src/screens/DiscoveryScreenV2/index.js

```diff
@@ -97,9 +115,22 @@ const DiscoveryScreenV2 = ({route}) => {
     };
 
     const fetchDiscoveryInitialDomains = async () => {
-      const initialData = await DiscoveryRepo.fetchInitialDiscoveryDomains();
-      FollowingAction.setFollowingDomain(initialData.suggestedDomains, followingDispatch);
-      DiscoveryAction.setDiscoveryInitialDomains(initialData.suggestedDomains, discoveryDispatch);
+      const initialData = await DiscoveryRepo.fetchInitialDiscoveryDomains(
+        50,
+        parseInt(domainPage.currentPage, 10)
+      );
+      setDomainPage({
+        currentPage: initialData.page,
+        totalPage: initialData.total_page
+      });
+      FollowingAction.setFollowingDomain(
+        [...discoveryData.initialDomains, ...initialData.suggestedDomains],
+        followingDispatch
+      );
+      DiscoveryAction.setDiscoveryInitialDomains(
+        [...discoveryData.initialDomains, ...initialData.suggestedDomains],
+        discoveryDispatch
+      );
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

Similar to the previous comment, this code block is repeating the same logic as in `fetchDiscoveryInitialUsers`. Consider using the generic function suggested above.
```diff
- const fetchDiscoveryInitialDomains = async () => {
-   const initialData = await DiscoveryRepo.fetchInitialDiscoveryDomains(
-     50,
-     parseInt(domainPage.currentPage, 10)
-   );
-   setDomainPage({
-     currentPage: initialData.page,
-     totalPage: initialData.total_page
-   });
-   FollowingAction.setFollowingDomain(
-     [...discoveryData.initialDomains, ...initialData.suggestedDomains],
-     followingDispatch
-   );
-   DiscoveryAction.setDiscoveryInitialDomains(
-     [...discoveryData.initialDomains, ...initialData.suggestedDomains],
-     discoveryDispatch
-   );
- };

+ // Use the generic function here
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->